test/python/algorithms/optimizers/test_optimizers.py

```diff
@@ -38,6 +38,7 @@
     P_BFGS,
     POWELL,
     SLSQP,
+    SNOBFIT,
```

Comment:
This line importing SNOBFIT should go too, to put things back as they were

Reply:
I saw it the moment I pushed the commit :D